lang/cs/Org.Apache.REEF.Bridge.JAR/Org.Apache.REEF.Bridge.JAR.csproj

```diff
@@ -23,7 +23,6 @@ under the License.
     <RootNamespace>Org.Apache.REEF.Bridge.JAR</RootNamespace>
     <ProjectGuid>{62905C7C-1A7E-4923-B78D-1BF42D7FAD40}</ProjectGuid>
     <AssemblyName>Org.Apache.REEF.Bridge.JAR</AssemblyName>
-	<OutputType>JAR</OutputType>
```

Comment:
That being said, the change is unrelated to the purpose of this PR and it would be good hygiene to remove it before merging.